src/diagonal.works/b6/ingest/features.go

```diff
@@ -64,7 +61,11 @@ func WrapFeature(f Feature, byID b6.FeaturesByID) b6.Feature {
 	case b6.FeatureTypeCollection:
 		return WrapCollectionFeature(f.(*CollectionFeature), byID)
 	case b6.FeatureTypeExpression:
-		return WrapExpressionFeature(f.(*ExpressionFeature))
+		// TO(andrew): should i leave this like this ? wrapping didn't do anything
```

Comment:
I think it's fine to leave it like this - at least it makes the decision that wrapping isn't necessary right now explicit.
At some point in the future, we'll have to wrap everything, because a single feature could have both an expression (which doesn't need wrapping) and geometry (that does). But then maybe wrapping moves from features to geometry (ie geometry values could need a World in order to actually get the S2 geometry by resolving the IDs).

Reply:
i am kind of hoping when i do areas / and smth smart w geometry expression / i lose physical feature / and then i just do wrapping for feature(i) / reference lookup by default / which will work for those that have references, and that's all i care about; but, let's see